addons/website/static/src/builder/plugins/form/utils.js

```diff
@@ -108,7 +107,7 @@ export function renderField(field, resetId = false) {
             });
         }
     }
-    const params = { field: { ...field }, defaultName: escape(field.string || _t("Field")) };
```

Comment:
Note: I don't see the point, it's used inside the `website.form_field` template as `t-att-` so it should be automatically escaped if necessary.

This is actually _wrong_ as it would double escape `&` chars, if there was any in `field.string` or translation of `Field`.